Gemfile

```diff
@@ -2,12 +2,10 @@ source 'https://rubygems.org'
 
 gem 'jekyll'
 gem 'json'
-gem 'redcarpet'
 gem 'open-uri-cached'
 gem 'jekyll-redirect-from'
 gem 'octopress-autoprefixer'
 
-gem 'rouge', '1.9'
 gem 'scss_lint', require: false
 
 gem 'jekyll-compose', group: [:jekyll_plugins]
```

Comment:
@maya Do you use `jekyll-compose` for anything?

Reply:
not sure what it's for or if I'm using it.

Reply:
Looks like its a tool for writing new posts and associating different states with them (published, unpublished, draft, etc). Looks like we are just adding new posts with regular jekyll way, so it should be safe to remove if we are looking to remove anything that's unused.

Reply:
Do you know what the `, group: [:jekyll_plugins]` does on the end of it?

Reply:
That just automatically includes the plugin in jekyll, it saves you a step of having to also include the name of the plugin in the `_config.yml`.